diff --git a/src/runtime.js b/src/runtime.js
index c904337956f35ae001239ad97007f0c72dc2c93f..8ca3f58f400ac20d357deb5105f339a17c5b0f45 100644
--- a/src/runtime.js
+++ b/src/runtime.js
@@ -13287,6 +13287,17 @@ module.exports = {
 
 
   function loadModuleData (moduleId) {
+    if (typeof window === 'undefined' && typeof require === 'function' && require('node:module').isBuiltin(moduleId)) {
+      return {
+        type: 'builtin',
+        package: moduleId,
+        id: moduleId,
+        // Using unprotected require
+        moduleInitializer: (_, module) => {
+          module.exports = require(moduleId);
+        },
+      }
+    }
     if (!moduleRegistry.has(moduleId)) {
       throw new Error(`no module registered for "${moduleId}" (${typeof moduleId})`)
     }
