diff --git a/src/index.js b/src/index.js
index a7e074a776c5e535101442bef16d130bf7eb0dbe..a9d8535e77c020cd9e3171807231454d715ef72b 100644
--- a/src/index.js
+++ b/src/index.js
@@ -146,9 +146,18 @@ function processOnePackageInLogicalTree(preferredPackageLogicalPathMap, resolve)
   return results
 }
 
+const REPLACEMENTS = { 'snaps-utils': '@metamask/snaps-utils' };
+
 function getPackageNameForModulePath(canonicalNameMap, modulePath) {
   const packageDir = getPackageDirForModulePath(canonicalNameMap, modulePath)
   if (packageDir === undefined) {
+    const matchesLocalPackage = modulePath.match(/\/packages\/([^\/]+)\/.+/);
+    const replacement = !modulePath.includes("node_modules") && matchesLocalPackage?.[1] && REPLACEMENTS[matchesLocalPackage[1]];
+  
+    if (replacement) {
+      return replacement;
+    }
+
     const relativeToRoot = path.relative(canonicalNameMap.rootDir, modulePath)
     return `external:${relativeToRoot}`
   }
