diff --git a/src/runtime.js b/src/runtime.js
index 97c9ddb9b76a2fe31cf9b202cb491f15232232ff..fe1cc43b933c77afdfd50e427a1989d5ed040005 100644
--- a/src/runtime.js
+++ b/src/runtime.js
@@ -10831,6 +10831,10 @@ function makeGetEndowmentsForConfig ({ createFunctionWrapper }) {
 function getPropertyDescriptorDeep (target, key) {
   let receiver = target
   while (true) {
+    // abort if this is the end of the prototype chain.
+    if (!receiver) {
+      return { prop: null, receiver: null }
+    }
     // support lookup on objects and primitives
     const typeofReceiver = typeof receiver
     if (typeofReceiver === 'object' || typeofReceiver === 'function') {
@@ -10845,10 +10849,6 @@ function getPropertyDescriptorDeep (target, key) {
       // eslint-disable-next-line no-proto
       receiver = receiver.__proto__
     }
-    // abort if this is the end of the prototype chain.
-    if (!receiver) {
-      return { prop: null, receiver: null }
-    }
   }
 }
 
@@ -11384,7 +11384,9 @@ module.exports = {
         const descriptors = Object.getOwnPropertyDescriptors(source)
         Object.values(descriptors).forEach(desc => {
           if ('get' in desc) {
+            try {
             Reflect.apply(desc.get, globalRef, [])
+            }catch{}
           }
         })
       })
