diff --git a/src/runtime.js b/src/runtime.js
index 1ae265438785cc893e1ff707a52729c96eed9dcc..fe50b39e6c9122f299d9f81ab45c5f3e8b47926f 100644
--- a/src/runtime.js
+++ b/src/runtime.js
@@ -12722,6 +12722,17 @@ module.exports = {
 
 
   function loadModuleData (moduleId) {
+    if (typeof window === 'undefined' && typeof require === 'function' && require('node:module').isBuiltin(moduleId)) {
+      return {
+        type: 'builtin',
+        package: moduleId,
+        id: moduleId,
+        // Using unprotected require
+        moduleInitializer: (_, module) => {
+          module.exports = require(moduleId);
+        },
+      }
+    }
     if (!moduleRegistry.has(moduleId)) {
       throw new Error(`no module registered for "${moduleId}" (${typeof moduleId})`)
     }
