diff --git a/dist/errors.d.ts b/dist/errors.d.ts
index 03c55f871a691de7b3f484c8984794cadde6b8a4..b21c15ae9066b5f0f92eafdc80e5d6309c8070b7 100644
--- a/dist/errors.d.ts
+++ b/dist/errors.d.ts
@@ -19,7 +19,7 @@ export declare function unauthorized(opts: UnauthorizedArg): import("@metamask/r
  * @param data - Optional data for context.
  * @returns The built error
  */
-export declare function methodNotFound(method: string, data?: DataWithOptionalCause): JsonRpcError<import("@metamask/rpc-errors/dist/utils").OptionalDataWithOptionalCause>;
+export declare function methodNotFound(method: string, data?: DataWithOptionalCause): JsonRpcError<import("@metamask/rpc-errors/dist/types/utils").OptionalDataWithOptionalCause>;
 declare type InvalidParamsArg = {
     message?: string;
     data?: DataWithOptionalCause;
