diff --git a/build/vite/utils.js b/build/vite/utils.js
index 3373a06d6b81f85c828bc9b4fa2fdd09e5a3b0ad..7098662a8cb9e72c6c24023b1988fa56d2190aa7 100644
--- a/build/vite/utils.js
+++ b/build/vite/utils.js
@@ -46,6 +46,11 @@ export async function getTemplate(options, env, spec, p = process) {
     catch (err) {
         log.error(`Failed to setup source-map-support: ${err.message}`);
     }
+
+    const mochaPath = await resolve('mocha', `${rootFileUrl}/node_modules`);
+    const mochaCSSHref = path.join(url.fileURLToPath(path.dirname(mochaPath)), 'mocha.css');
+    const mochaJSSrc = path.join(url.fileURLToPath(path.dirname(mochaPath)), 'mocha.js');
+
     return /* html */ `
     <!doctype html>
     <html>
@@ -78,7 +83,8 @@ export async function getTemplate(options, env, spec, p = process) {
                     return mod
                 }
             </script>
-            <script type="module" src="/node_modules/mocha/mocha.js"></script>
+            <link rel="stylesheet" href="${mochaCSSHref}">
+            <script type="module" src="${mochaJSSrc}"></script>
             ${sourceMapScript}
             <script type="module">
                 ${sourceMapSetupCommand}
