diff --git a/default-input.js b/default-input.js
index 7d859a0d9b105eafb02c047627da7698412057c0..fcbfa1d31a2e369183a5fbd233d058ea76667970 100644
--- a/default-input.js
+++ b/default-input.js
@@ -105,7 +105,7 @@ if (!package.main) {
       else
         f = f[0]
 
-      var index = f || 'index.js'
+      var index = config.get('init.main') || f || 'src/index.js'
       return cb(null, yes ? index : prompt('entry point', index))
     })
   }
