diff --git a/src/kernelTemplate.js b/src/kernelTemplate.js
index de8d73048b7bb7c4e74009b5c85ad919fe197ef0..9a70b7afd5ae343f01c49b564622393a3dae6244 100644
--- a/src/kernelTemplate.js
+++ b/src/kernelTemplate.js
@@ -60,6 +60,8 @@
       errorTaming: 'unsafe',
       // shows the full call stack
       stackFiltering: 'verbose',
+      // prevent issue when dealing with the "override mistake"
+      overrideTaming: 'severe',
     }

     lockdown(lockdownOptions)
