diff --git a/src/index.js b/src/index.js
index db802666fb1126700dc604008d51c0f152b6cb37..7b45c121270ad209c8c9f0b36b4307b6125b333f 100644
--- a/src/index.js
+++ b/src/index.js
@@ -72,7 +72,7 @@ function plugin(browserify, pluginOpts) {
         createModuleInspectorSpy({
           policyOverride,
           // no builtins in the browser (yet!)
-          isBuiltin: () => false,
+          isBuiltin: (name) => browserify._options.node && isBuiltin(name),
           // should prepare debug info
           includeDebugInfo: configuration.writeAutoPolicyDebug,
           // write policy files to disk
@@ -321,7 +321,7 @@ function validatePolicy(policy) {
   Object.entries(policy.resources).forEach(([, packageOpts]) => {
     const packageOptions = Object.keys(packageOpts)
     const packageEntries = Object.values(packageOpts)
-    const optionsWhitelist = ['globals', 'packages']
+    const optionsWhitelist = ['globals', 'packages', 'builtin']
     const valuesWhitelist = [true, 'write']
 
     if (
@@ -330,7 +330,7 @@ function validatePolicy(policy) {
       )
     ) {
       throw new Error(
-        "LavaMoat - Unrecognized package options. Expected 'globals' or 'packages'"
+        "LavaMoat - Unrecognized package options. Expected 'globals', 'packages' or 'builtin'"
       )
     }
 
