diff --git a/node_modules/ts-auto-guard/lib/index.js b/node_modules/ts-auto-guard/lib/index.js
index 7294c22..70616eb 100644
--- a/node_modules/ts-auto-guard/lib/index.js
+++ b/node_modules/ts-auto-guard/lib/index.js
@@ -425,7 +425,7 @@ function typeConditions(varName, type, addDependency, project, path, arrayDepth,
         return eq(varName, 'null');
     }
     if (type.getText() === 'any' || type.getText() === 'unknown') {
-        return null;
+        return 'true';
     }
     if (type.getText() === 'never') {
         return typeOf(varName, 'undefined');
